### PR TITLE
docs: improve accessibility of form-validation example

### PR DESCRIPTION
**Pull request by kapunahelewong**
Fixes contrast issues.
Adds padding to labels.
Changes label to explicitly state intent.
Adds explanatory text about disabled button.
Adds aria-disabled=true to disabled button.
Changes markup to make it more semantic.
Updates CSS.

## Before

Screen Shot 2021-03-29 at 10 20 39 AM

## After

Screen Shot 2021-03-29 at 10 20 27 AM

### CODE REVIEW

File: aio/content/examples/form-validation/src/app/reactive/hero-form-reactive.component.html [hunk 1]
```diff
@@ -63,8 +63,11 @@ <h1>Reactive Form</h1>
         </div>
       </div>
 
-      <button type="submit" class="btn btn-default"
-             [disabled]="heroForm.invalid">Submit</button>
+      <p>Complete the form to enable the Submit button.</p>
+      <button type="submit"
+              class="btn btn-default"
+              aria-disabled=true
```
Comment (destus90) [inline aio/content/examples/form-validation/src/app/reactive/hero-form-reactive.component.html:69]: Due to this change the `aria-disabled` attribute is never updated even if the form becomes valid? 
```
[attr.aria-disabled]="heroForm.invalid"
```

Comment (kapunahelewong) [reply aio/content/examples/form-validation/src/app/reactive/hero-form-reactive.component.html:69]: Thank you so much, @destus90! So the whole button should be like this?

```
      <button type="submit"
              class="btn btn-default"
              [attr.aria-disabled]="heroForm.invalid">Submit</button>
```

Comment (destus90) [reply aio/content/examples/form-validation/src/app/reactive/hero-form-reactive.component.html]: @kapunahelewong 
Yeah, looks good to me 🤗

Comment (destus90) [reply aio/content/examples/form-validation/src/app/reactive/hero-form-reactive.component.html]: @kapunahelewong 
I just noticed you removed `disabled` property binding on the button in favor of `aria-disabled` attribute. You shouldn't have done it. You should save both. This is the correct syntax:
```
<button type="submit" class="btn btn-default" [disabled]="heroForm.invalid" [attr.aria-disabled]="heroForm.invalid">
  Submit
</button>
```

Comment (kapunahelewong) [reply aio/content/examples/form-validation/src/app/reactive/hero-form-reactive.component.html]: Fixed! Thank you, @destus90!

**Review by AndrewKushnir — APPROVED** [review]

File: aio/content/examples/form-validation/src/app/reactive/hero-form-reactive.component.html [hunk 2]
```diff
@@ -63,8 +63,11 @@ <h1>Reactive Form</h1>
         </div>
       </div>
 
-      <button type="submit" class="btn btn-default"
-             [disabled]="heroForm.invalid">Submit</button>
+      <p>Complete the form to enable the Submit button.</p>
+      <button type="submit"
+              class="btn btn-default"
+              [disabled]="heroForm.invalid"
+              [attr.aria-disabled]="heroForm.invalid">Submit</button>
```
Comment (jelbourn) [inline aio/content/examples/form-validation/src/app/reactive/hero-form-reactive.component.html:70]: `aria-disabled` shouldn't be necessary here

File: aio/content/examples/form-validation/src/app/template/hero-form-template.component.html [hunk 3]
```diff
@@ -35,42 +35,48 @@ <h1>Template-Driven Form</h1>
...
-        <div *ngIf="power.errors && power.touched" class="alert alert-danger">
+        <div *ngIf="power.errors && power.touched" class="alert">
           <div *ngIf="power.errors.required">Power is required.</div>
         </div>
       </div>
 
-      <button type="submit" class="btn btn-default"
+      <p>Complete the form to enable the Submit button.</p>
+      <button type="submit"
+              aria-disabled=true
```
Comment (jelbourn) [inline aio/content/examples/form-validation/src/app/template/hero-form-template.component.html:77]: Same for `aria-disabled` here

**Review by aikithoughts — APPROVED** [review]
LGTM
Reviewed-for: global-docs-approvers

**Review by aikithoughts — APPROVED** [review]
Reviewed-for: global-docs-approvers

**Pull request closed without merging**